WcaOnRails/app/views/competitions/_nav.html.erb

```diff
@@ -187,6 +187,12 @@
         <% end %>
       <% end %>
 
+      <% if @competition.user_should_post_competition_results?(current_user) %>
+        <%= alert :warning, note: true do %>
+          Please submit results for this competition! You can upload them <%= link_to "here", submit_results_edit_path(@competition) %>.
```

Comment:
Shouldn't be "the results" instead of just "results"?

Reply:
Speaking as an English speaker, both "results" and "the results" feel equally natural to me here. I'll let @LinusFresz decide what he prefers =)